chichiwang/exercise-js
# Build Documentation page section: NPM Scripts

## Description
Linked to Issue: #65
Build out the section "NPM Scripts" on the `/documentation` page. In doing so, modify the component `CopyButton` to accept a `className` prop.

## Changes
* Grammar fix in the "NPM Scripts" section of the `docs/README.md` file
* Update `/pages/Documentation/index.tsx` to add a section: NPM Scripts
  * Additionally style the Contents section
* Accept a prop `className` in component `pages/components/CopyButton/index.tsx`
  * Style to not display component for print media

## Steps to QA
* Pull down and switch to this branch
* Run the application locally and verify it looks good in browser at the `/documentation` route
  * Verify the content looks good at all screen resolutions
  * Verify the content looks good when printing (either through print emulation or print preview)
  * Verify the copy button on each NPM script copies the npm script command correctly
* Verify the changes to `docs/README.md` look good
